src/main/java/com/comphenix/protocol/wrappers/AdventureComponentConverter.java

```diff
@@ -0,0 +1,59 @@
+/**
+ *  ProtocolLib - Bukkit server library that allows access to the Minecraft protocol.
+ *  Copyright (C) 2015 dmulloy2
+ *
+ *  This program is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU General Public License as published by the Free Software Foundation; either version 2 of
+ *  the License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with this program;
+ *  if not, write to the Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ *  02111-1307 USA
+ */
+package com.comphenix.protocol.wrappers;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.serializer.gson.GsonComponentSerializer;
+
+/**
+ * Utility class for converting between the Adventure API Component and ProtocolLib's wrapper
+ * <p>
+ * Note: The Adventure API Component is not included in CraftBukkit, Bukkit or Spigot and is only present in PaperMC.
+ * @author LOOHP
```

Comment:
I appreciate the documentation, but this looks like the code from https://github.com/PaperMC/Paper/issues/5220#issuecomment-783363243 so I'd probably remove the author tag